src/modules/contactUs/contact-us.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Body, Controller, HttpException, HttpStatus, Post } from '@nestjs/common';
+import { ContactService } from './contact-us.service';
+import { ContactUsDto } from './dto/contact-us.dto';
+
+@Controller('api/v1/contact-us')
+export class ContactController {
+  constructor(private readonly contactService: ContactService) {}
```

Comment:
I believe this route is a public route. If yes add the decorator @skipAuth() to this endpoint

Reply:
I have